visual-js/visual-snapshots/test/app/pdf-handler.spec.ts

```diff
@@ -55,17 +70,71 @@ describe("pdf-handler", () => {
       };
 
       await handler.handle(
-        [path.join(__dirname(import.meta), "../files/1.pdf")],
+        [path.join(__dirname(import.meta), "../files/")],
         params
       );
 
       expect(createBuildMock).toHaveBeenCalledWith(params);
+      expect(mockFileExtractor.calls()).toEqual([
+        {
+          globOrDirs: [path.join(__dirname(import.meta), "../files/")],
+          dirGlob: "*.pdf",
+        },
+      ]);
+      expect(logged).toEqual([
+        {
+          level: 30,
+          msg: "Found 2 PDF files to process: /absolute/path/to/files/1.pdf, /absolute/path/to/files/2.pdf",
+        },
+      ]);
     });
 
     it("should not create a build when buildId is passed", async () => {
+      mockFileExtractor.setFilesToReturn(["/absolute/path/to/files/1.pdf"]);
+
+      const handler = new PdfCommandHandler(
+        visualSnapshotsApi,
+        pdfSnapshotUploaderMock,
+        mockFileExtractor,
+        logger
+      );
+
+      const params: PdfCommandParams = {
+        suiteName,
+        testName,
+        snapshotName,
+        buildId,
+        concurrency: 1,
+      };
+
+      await handler.handle(
+        [path.join(__dirname(import.meta), "../files/1.pdf")],
+        params
+      );
+
+      expect(createBuildMock).not.toHaveBeenCalled();
+      expect(mockFileExtractor.calls()).toEqual([
+        {
+          globOrDirs: [path.join(__dirname(import.meta), "../files/1.pdf")],
+          dirGlob: "*.pdf",
+        },
+      ]);
+      expect(logged).toEqual([
+        {
+          level: 30,
+          msg: "Found 1 PDF file to process: /absolute/path/to/files/1.pdf",
+        },
+      ]);
```

Comment:
Do you think `expect(logged).toMatchSnapshot()` would be a better idea?

Reply:
As this is just a single log, I think this is simpler this way but I am fine with using snapshots if that's your preference.